packages/embed/src/utils/set-max-height.ts

```diff
@@ -0,0 +1,11 @@
+import { isMobile } from './mobile'
+
+export const setMaxHeight = (element: HTMLElement) => {
+  const mediaQuery = window.matchMedia('(max-width: 480px)')
+
+  if (isMobile() && mediaQuery.matches) {
```

Comment:
please test this condition in a unit test

Reply:
yeah I wanted to, but it's hard to unit test this code since it uses height and window. The mobile util is already tested, let me see if I can add some test to this file